libs/transfer-journey/shared-data/project.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "transfer-journey-shared-data",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "projectType": "library",
+  "sourceRoot": "libs/transfer-journey/shared-data/src",
```

Comment:
I thought that we are going to have a separate `shared` lib.
Is this just for the `transfer-journey` or is it being shared with other journeys as well?

Reply:
This data is not relevant to any other Journey. So why do we need to create a shared folder for other Journeys.